view-samples/shopping-cart/shopping-cart.json

```diff
@@ -0,0 +1,314 @@
+                      "align-items": "center"
+                    },
+                    "children": [
+                      {
+                        "elmType": "span",
+                        "style": {
+                          "font-size": "16px",
+                          "margin-right": "4px"
+                        },
+                        "txtContent": "$"
+                      },
+                      {
+                        "elmType": "span",
+                        "style": {
+                          "font-size": "48px"
+                        },
+                        "txtContent": "[$DollarValue]"
+                      },
+                      {
+                        "elmType": "span",
+                        "style": {
+                          "font-size": "16px"
+                        },
+                        "txtContent": "[$CentsValue]"
+                      }
+                    ]
+                  }
+                ]
+              },
+              {
+                "elmType": "div",
+                "style": {
+                  "font-size": "16px",
+                  "color": "#666666",
+                  "display": "flex",
+                  "flex-direction": "column",
+                  "padding": "10px",
+                  "align-items": "flex-end"
+                },
+                "children": [
+                  {
+                    "elmType": "div",
+                    "style": {
+                      "text-decoration": "line-through",
+                      "color": "#999999",
+                      "margin-bottom": "4px"
+                    },
+                    "txtContent": "='Was $' + [$OldPrice]"
+                  },
+                  {
+                    "elmType": "div",
+                    "style": {
+                      "color": "#d32f2f"
+                    },
+                    "txtContent": "='Save $' + [$CalculatedPrice]"
+                  }
+                ]
+              }
+            ]
+          },
+          {
+            "elmType": "div",
+            "style": {
+              "display": "flex",
+              "align-items": "center",
+              "justify-content": "space-between",
+              "margin-bottom": "12px",
+              "width": "100%"
+            },
+            "children": [
+              {
+                "elmType": "span",
+                "style": {
+                  "font-size": "16px",
+                  "color": "#ff6347",
+                  "font-weight": "bold"
+                },
+                "txtContent": "=if([$Shipping] > 0, 'Shipping: $' + [$Shipping], 'Free Shipping')"
+              },
+              {
+                "elmType": "button",
+                "customRowAction": {
+                  "action": "setValue",
+                  "actionInput": {
+                    "AddedToCart": "=if([$AddedToCart] == true, false, true)",
+                    "ApprovalActionBy": "@me"
```

Comment:
If updating `ApprovalActionBy`, I think the `ApprovalActionBy` column needs to be included in the view. So, if you are going to include the update of `ApprovalActionBy` in this sample, please include `ApprovalActionBy` in the View requirements in the README. (If you do not need to update ApprovalActionBy, please remove the program that updates ApprovalActionBy from the JSON.)

Reply:
Removed from JSON